python/dasclient/dasclient.py

```diff
@@ -391,20 +436,19 @@ def get_subject_tracks(self, subject_id='', start=None, end=None):
 
         return self._get(path='subject/{0}/tracks'.format(subject_id), params=p)
 
-    def get_subjects(self):
+    def get_subjects(self, params = {}):
```

Comment:
Instead of assigning a literal dict, give it a default of None and resolve in code. _In your case this might not matter much, but in general it's best to not have a mutable object as default _

Instead try this:
```
    def get_subjects(self, params=None):
        params = params or {}
        ...
```